packages/fxa-auth-server/lib/account-events.ts

```diff
@@ -68,11 +81,35 @@ export class AccountEventsManager {
         if (!value) delete emailEvent[key as keyof typeof emailEvent];
       }
 
-      await this.usersDbRef.doc(uid).collection('events').add(emailEvent);
+      await this.usersDbRef?.doc(uid).collection('events').add(emailEvent);
       this.statsd.increment('accountEvents.recordEmailEvent.write');
     } catch (err) {
       // Failing to write to events shouldn't break anything
       this.statsd.increment('accountEvents.recordEmailEvent.error');
     }
   }
+
+  /**
+   * Record a security event for the user. This is based on our security events
+   * that are stored in MySQL.
+   *
+   * @param db - auth db
+   * @param message - message
+   */
+  public recordSecurityEvent(db: any, message: SecurityEvent) {
```

Comment:
I kind of went back and forth with whether or not to pass in the db or have it as part of the constructor. Doing the latter was going to have alot of testing churn. Also considering we might do  https://mozilla-hub.atlassian.net/browse/FXA-6268.

Reply:
Makes sense to me. Random nit,  can we have a type for db? It can even just be an inline type for a db that supports the securityEvent method?